template.tpl

```diff
@@ -1124,6 +1124,48 @@ ___TEMPLATE_PARAMETERS___
                           "help": "Define list of URL regex patterns to capture. By default any URL is captured. (comma separated list)."
                         },
                         "isUnique": false
+                      },
+                      {
+                        "param": {
+                          "type": "TEXT",
+                          "name": "responseHeaders",
+                          "displayName": "Response Headers",
+                          "help": "The response headers to capture, comma separated list. e.g.: \"Content-Type,Set-Cookie\"",
+                          "simpleValueType": true,
+                          "defaultValue": "access-control-allow-origin, access-control-allow-credentials, access-control-expose-headers, access-control-max-age, access-control-allow-methods, access-control-allow-headers, accept-patch, accept-ranges, age, allow, alt-svc, cache-control, connection, content-disposition, content-encoding, content-language, content-length, content-location, content-md5, content-range, content-type, date, delta-base, etag, expires, im, last-modified, link, location, permanent, p3p, pragma, proxy-authenticate, public-key-pins, retry-after, server, status, strict-transport-security, trailer, transfer-encoding, tk, upgrade, vary, via, warning, www-authenticate, x-b3-traceid, x-frame-options"
```

Comment:
This is just a reminder that if we are updating default values we should also update it here. I can see why we use default value here for simplicity. `responseHeaders?: string[] | boolean;` is a union type. It's equivalent to true when keeping the default value and it's equivalent to false when passing an empty string